.vscode/extensions.json

```diff
@@ -1,17 +1,7 @@
 {
-	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
-	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
-
-	// List of extensions which should be recommended for users of this workspace.
 	"recommendations": [
 		"ms-python.vscode-pylance",
 		"ms-python.python",
-		"streetsidesoftware.code-spell-checker",
-		"njpwerner.autodocstring",
-		"kevinrose.vsc-python-indent"
```

Comment:
Could you undo these tweaks please? I'm recommending the spell checker since it's pretty important to ensure that things are spelt correctly within the documentation. The Python indent one can probably be removed, since it's just a quality of life thing (although I'll sort that out in a separate PR).